config/src/tests/file_subscription/my.cfg

```diff
@@ -1 +1 @@
-myField "foobar"
+myField "foo"
```

Comment:
Can't remember ever touching this file or adding it to the commit, so not sure where this diff came from 🤔 IDE must have changed it somehow and snuck it under the radar. Will remove.